app/questionnaire/router.py

```diff
@@ -71,20 +76,23 @@ def get_next_location_url(self, location, routing_path, return_to=None):
         Get the next location in the section. If the section is complete, determine where to go next,
         whether it be a summary, the hub or the next incomplete location.
         """
-
-        if return_to:
-            is_section_complete = self._progress_store.is_section_complete(
-                location.section_id, location.list_item_id
-            )
-            if is_section_complete and return_to == "section-summary":
+        is_section_complete = self._progress_store.is_section_complete(
```

Comment:
This doesn't need to be assigned as it's only used once
```suggestion
```

Reply:
Done, thanks!👍